libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/model/ManeuverViewOptions.kt

```diff
@@ -1,17 +1,35 @@
 package com.mapbox.navigation.ui.maneuver.model
 
+import androidx.annotation.ColorRes
+import androidx.annotation.StyleRes
+import com.mapbox.navigation.ui.maneuver.R
+import com.mapbox.navigation.ui.maneuver.view.MapboxLaneGuidance
 import com.mapbox.navigation.ui.maneuver.view.MapboxPrimaryManeuver
 import com.mapbox.navigation.ui.maneuver.view.MapboxSecondaryManeuver
+import com.mapbox.navigation.ui.maneuver.view.MapboxStepDistance
 import com.mapbox.navigation.ui.maneuver.view.MapboxSubManeuver
+import com.mapbox.navigation.ui.maneuver.view.MapboxTurnIconManeuver
 
 /**
  * Specifies options for rendering different components in a [Maneuver].
  *
+ * @param maneuverBackgroundColor to style main maneuver background color
+ * @param subManeuverBackgroundColor to style sub maneuver background color
+ * @param upcomingManeuverBackgroundColor to style upcoming maneuver background color
+ * @param turnIconManeuver to style the [MapboxTurnIconManeuver] turn icon colors
+ * @param laneGuidanceTurnIconManeuver to style the [MapboxLaneGuidance] turn icon colors
+ * @param stepDistanceOptions to style [MapboxStepDistance]
  * @param primaryManeuverOptions to style [MapboxPrimaryManeuver]
  * @param secondaryManeuverOptions to style [MapboxSecondaryManeuver]
  * @param subManeuverOptions to style [MapboxSubManeuver]
  */
 class ManeuverViewOptions private constructor(
+    @ColorRes val maneuverBackgroundColor: Int,
+    @ColorRes val subManeuverBackgroundColor: Int,
+    @ColorRes val upcomingManeuverBackgroundColor: Int,
+    @StyleRes val turnIconManeuver: Int,
+    @StyleRes val laneGuidanceTurnIconManeuver: Int,
+    val stepDistanceOptions: ManeuverStepDistanceOptions,
```

Comment:
This seems to fit better with the rest of the options

```suggestion
    @StyleRes val stepDistanceTextAppearance: Int,
```

Reply:
Done.